src/main/java/com/marek/onlinebookstore/controller/BookController.java

```diff
@@ -33,4 +36,16 @@ public BookDto createBook(@RequestBody
                               CreateBookRequestDto createBookRequestDto) {
         return bookService.save(createBookRequestDto);
     }
+
+    @PostMapping("/{id}")
```

Comment:
It should be like this:
```suggestion
    @PutMapping("/{id}")
```